cmd/device-util/driver.go

```diff
@@ -0,0 +1,159 @@
+//go:build windows
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+const pollTick = 5 * time.Millisecond
+
+// flag names
+
```

Comment:
Extra line?

Reply:
if its written without it, like:
```go
// flag names
const( 
  //...
)
```
Then `flag names` becomes the doc comment for all the flag names. I just wanted to section off those `const`s from `pollTick`.

Reply:
Maybe just change the comment to something like `// Names of command line flags.`, then it's fine if it shows up as a doc comment.

Alternately it's pretty clear they are flags, so we could just remove the comment.